corehq/form_processor/backends/sql/ledger.py

```diff
@@ -77,6 +77,7 @@ def _lazy_original_balance():
         ledger_value.track_create(transaction)
         # only do this after we've created the transaction otherwise we'll get the wrong delta
         ledger_value.balance = new_ledger_values.balance
+        ledger_value.last_modified = stock_trans.timestamp
```

Comment:
It's been a long-backlogged item to convert to using server times for ledger dates and order logic. If it's an easy change to make now we should do it - I don't think it will break anything.